static/css/components/user.css

```diff
@@ -46,3 +46,11 @@ p#friendsText {
 	color: #FCFCFC;
 	font-size: 2em;
 }
+
+img#achieveImage {
```

Comment:
Empty CSS can be removed

Reply:
Won't it break the code below?

Reply:
Nope, the style will work just for the hover specifically, no need to select the whole element if it won't be styled